ability_to_view_the_File_related_list_in_the_Opportunity_records.feature

```diff
@@ -0,0 +1,15 @@
+Feature:View the file related list in the Opportunity records
+
+  Scenario Outline: ability to view the File related list in the Opportunity records
+
+    Given I am on the "<application>" Application as a "<fg_user>"
```

Comment:
Please use the existing SD's 'Given I am logged in as a "<user> user"'
In the next step please write the generic SD to  select the "application"